alembic.rst

```diff
@@ -0,0 +1,84 @@
+Alembic
+=======
+Alembic is a database migration tool for usage with SQLAlchemy Database. The full documentation can be found here: 
+In invenio the alembic environment is already defined in invenio-db
+
+Creating new revisions
+----------------------
+
+When working with an invenio module that already has alembic revisions, after modifying a model we can create a new alembic revision with:
+
+.. code-block:: console 
+
+    $ invenio alembic revision "Added some column." --path alembic_directory_path
+
+A short message describing the changes is required and the path parameter should point to the alembic directory of the module. If the path is not given the new revision will be placed in the invenio_db/alembic directory.
+
+The directories accepted are registered by the invenio_db.alembic entrypoints.
+If the DB has not been updated to show the last changes, there is some alembic revision that needs to be applied by:
+
+.. code-block:: console 
+
+    $ invenio alembic upgrade heads
```

Comment:
Shouldn't we start by running this before creating the new script so that everything is up to date? The user can normally run this without worrying about breaking anything as long as he didn't create his migration script.